src/main/java/org/jacksonlaboratory/model/dto/SearchDto.java

```diff
@@ -20,4 +21,17 @@ public List<OntologyTerm> getTerms() {
 	public int getTotalCount() {
 		return totalCount;
 	}
+
+	@Override
+	public boolean equals(Object o) {
```

Comment:
A fellow implementor of equals and hashcode, I autogenerate assume IntelliJ does too.